.gitmodules

```diff
@@ -131,3 +131,6 @@
 [submodule "generators/ibex"]
 	path = generators/ibex
 	url = https://github.com/ucb-bar/ibex-wrapper
+[submodule "generators/FFTGenerator"]
```

Comment:
To maintain consistency, name this `fft-generator`

Reply:
+1

Reply:
This repo should exist within `ucb-bar`